AnimeshKumarRai/task4.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>task4</title>
+    <link rel="stylesheet" href="/Web/Task/web-development-task1/assets/css/navbar.css">
```

Comment:
Seems css link is dependent on the browsing location.
I suggest to change it to
```suggestion
    <link rel="stylesheet" href="../assets/css/navbar.css">
```